### PR TITLE
add guide for ArProfile

### PR DESCRIPTION
**Pull request by PBillingsby**

### CODE REVIEW

File: docs/src/guides/arprofile.md [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+Optional argument options can be passed into the constructor. Here is the default options:
+```js:no-line-numbers
+const opts = {
+  cacheIsActivated: true,
+  cacheSize: 100,
+  cacheTime: 60
+};
+```
+
```
Comment (twilson63) [inline docs/src/guides/arprofile.md:38]: Maybe a callout container to define what caching does?

File: docs/src/guides/arprofile.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+    "bio": "test account by cromatikap\nUPDATE",
+    "email": "",
+    "links": {
+      "github": "cromatikap",
+      "twitter": "cromatikap"
+    },
+    "wallets": {}
+  }
+}
+```
```
Comment (twilson63) [inline docs/src/guides/arprofile.md:124]: Maybe a summary with links to the arprofile site

**Review by twilson63 — COMMENTED** [review]
Made some comments, Looks great!

**Comment by PBillingsby** [comment]
@twilson63 added some more based on your comments!

**Review by twilson63 — APPROVED** [review]
LGTM

**Pull request merged**